analyzer/codechecker_analyzer/analyzers/clangsa/analyzer.py

```diff
@@ -336,7 +379,9 @@ def construct_config_handler(cls, args, context):
         if 'ctu_phases' in args:
             handler.ctu_dir = os.path.join(args.output_path,
                                            args.ctu_dir)
-
+            handler.ctu_on_demand = \
+                'ctu_ast_creation' in args and \
+                args.ctu_ast_loading_mode == 'parsed-on-demand'
```

Comment:
```suggestion
                args.ctu_ast_creation == 'parsed-on-demand'
```

Reply:
Thanks for catching this, I have updated both occurences with ctu_ast_mode now.